src/main/java/ai/elimu/web/content/MainContentController.java

```diff
@@ -119,7 +119,7 @@ public String handleRequest(
         
         model.addAttribute("letterCount", letterDao.readCount());
         model.addAttribute("soundCount", soundDao.readCount());
-        model.addAttribute("letterSoundCorrespondenceCount", letterSoundDao.readCount());
+        model.addAttribute("letterSoundCount", letterSoundDao.readCount());
```

Comment:
This change also needs to be reflected in the controller's frontend file at `src/main/webapp/WEB-INF/jsp/content/main.jsp`.

<kbd>![](https://github.com/user-attachments/assets/613c0c76-21d2-4b61-960d-2622a0b679b5)</kbd>